modules/client/test/integration/multisig-client/methods.test.ts

```diff
@@ -311,12 +310,10 @@ describe("Client Multisig", () => {
       const ctxPlugin = ContextPlugin.fromContext(ctx);
       const client = new MultisigClient(ctxPlugin);
       const limit = 5;
-      const status = ProposalStatus.EXECUTED;
       const params: IProposalQueryParams = {
         limit,
         sortBy: ProposalSortBy.CREATED_AT,
         direction: SortDirection.ASC,
-        status,
       };
```

Comment:
Why removing this?

Reply:
Currenly on subgraph we dont have any proposal thats executed so the test will fail 😓

Reply:
Can't we ensure to add some?  or changing the filter and checking that the results change from "something" to "nothing" ?

Reply:
I would't make sense, because tomorrow we could have an executed proposal and break the tests again :(
Either wee automate the creation of daos and proposals each time we deploy the contracts or we start mocking the subgraph